app/templates/people/person/positions/minister/index.hbs

```diff
@@ -0,0 +1,130 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuToolbar>
+        <AuToolbarGroup>
+          <AuHeading>
+            Positie:
+            {{@model.minister.ministerPosition.function.label}},
+            <br />
+            {{@model.minister.ministerPosition.worshipService.name}}
+          </AuHeading>
+        </AuToolbarGroup>
+        <AuToolbarGroup>
+          {{! TODO: Replace this with the "edit" route once it is created }}
+          <AuLink @route="people.person.positions.minister" @skin="button-secondary" @icon="pencil"
+            @iconAlignment="left">
+            Bewerk
+          </AuLink>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuHeading @level="2" @skin="3" class="au-u-margin-top-tiny">
+        {{@model.person.givenName}}
+        {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Positie</:title>
+      <:card as |Card|>
+        <Card.Grid as |Item|>
+          <Item>
+            {{! TODO: Make it variable to also have Bedienaar}}
```

Comment:
Since this page is now dedicated to minister positions, I think this comment can simply be removed?

Reply:
(Be sure to also remove this in the mandatory version. I can't leave a comment since the PR doesn't change that file.)